### DISCUSSION TITLE
A reason why Ansel and Darktable aren't ready for the professional photographers

### DISCUSSION THREAD

**Discussion by mmoqui** [aurelienpierreeng/ansel#101] [General]
I found on Youtub a video (in French, sorry) comparing in perf the import and the export of images in the case of a professional usage. The author conclusion is whether the time of import or export can be important, the true difference in fact is what it is between the import and the export (the development of images). For him, Darktable has great capabilities but also has too lot of frictions for a professional usage (he explains why).

In my understanding, the frictions in Darktable for a professional usage were one of the reasons of why Aurelien Pierre forked Darktable.

I've found the video interesting because it is the point of view of a professional photographer about Darktable (he is/was a user of Darktable).

The video is here (sorry it's in French): https://www.youtube.com/watch?v=QEMMhoivk88

**Comment by aurelienpierre** [comment]
As a starting point, let's not talk about the professional photographer.

Photojournalists, wedding and fine-art photographers are all professionals with different needs, since they work with different volumes of pictures for different media with different deadlines and final quality expectations.

**Reply by photomf** [reply]
mah !  am a pro photojournalist based in Italy And I've been following the Ansel Photo project for a while now, partly because I work on two computers running Linux, To be honest, previous versions of the program on my PCs had issues that limited their usability (sudden freezes and slowdowns, and some tools—like the cropping tool—that didn’t work). The latest version I downloaded works well, and I’m currently using it to edit short photo shoots and a few preseason soccer matches in Italy.

**Comment by lilygthb** [comment]
I feel that at the moment Ansel is mature enough to be used as primary development software. In terms of organization, UX and graphic design has made gigantic progresses from some months ago, I tried but was still too work in progress.
Now, while still not finished, works well enough that I've switched from Darktable.

**Reply by photomf** [reply]
That's true... I'm still learning how to use it right now, but based on the tests I've done, I can say that the file I'm able to produce is consistent with what I get from the other open-source software I normally use. At the moment, I'm not yet very proficient with the workflow, due to the interface, but that's my own limitation.

**Comment by lilygthb** [comment]
this video by @aurelienpierre is basically a complete walkthrough Ansel, how it work and why it works better than Darktable.
https://www.youtube.com/watch?v=1R7gdfJVH9Q
It's very long, but dense with information, you can split it in sessions.
I watched once very quickly, but I'm watching again for the in depth explaination of inner working of Ansel.
